### PR TITLE
Generate parameters for Joint State Broadcaster

### PR DESCRIPTION
**Pull request by tylerjw**
Signed-off-by: Tyler Weaver <tyler@picknik.ai>

generate_parameter_library for joint_state_broadcaster

### CODE REVIEW

**Review by jaron-l — APPROVED** [review]
Looks really clean.

**Review by destogl — APPROVED** [review]
Thanks this looks very good. Can we also add comments/documentation for each parameter?

**Comment by tylerjw** [comment]
> Thanks this looks very good. Can we also add comments/documentation for each parameter?

Would you mind helping me do that? I have checked the box so you can push into any of these PRs.

File: ros2_controllers-not-released.rolling.repos [hunk 1]
```diff
@@ -1 +1,5 @@
 repositories:
+  generate_parameter_library:
+    type: git
+    url: https://github.com/picknikrobotics/generate_parameter_library.git
+    version: main
```
Comment (destogl) [inline ros2_controllers-not-released.rolling.repos:5]: With this change, mail builds should fail until next sync.
```suggestion
```

File: joint_state_broadcaster/src/joint_state_broadcaster_parameters.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+joint_state_broadcaster:
+  use_local_topics: {
+    type: bool,
+    default_value: false,
+  }
+  joints: {
+    type: string_array,
+    default_value: [],
+  }
```
Comment (bmagyar) [inline joint_state_broadcaster/src/joint_state_broadcaster_parameters.yaml:9]:
```suggestion
  }
  extra_joints: {
    type: string_array,
    default_value: [],
  }
```

**Pull request merged**